src/sql/workbench/contrib/notebook/browser/markdownToolbarActions.ts

```diff
@@ -128,7 +147,7 @@ export class MarkdownTextTransformer {
 			case MarkdownButtonType.ORDERED_LIST:
 				return '1. ';
 			case MarkdownButtonType.IMAGE:
-				return '![ALT TEXT](';
+				return '![';
```

Comment:
Some fun potential here for what Github does with apparently detecting if it's a link or not and then inserting the selected text into either the alt text portion or the link portion depending. But that's definitely just a "nice-to-have".

This seems ok, although I'd be curious whether people generally prefer to create the text first and then link or create the link and then set the text (I usually do the latter so what you had before would be better for me)